prep-css-flexbox/align-items.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>CSS Flexbox - align-items</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
+    div#card-deck {
+      display: flex;
+      flex-basis: calc(100% / 3);
```

Comment:
Hi Lea, `flex-basis` goes on flex item elements, so try moving this to a rule set for the `.card-wrapper` class.